openedx/features/enterprise_support/api.py

```diff
@@ -268,7 +293,7 @@ def consent_needed_for_course(user, course_id):
     return consent_necessary_for_course(user, course_id)
 
 
-def get_enterprise_consent_url(request, course_id, user=None, return_to=None):
+def get_enterprise_consent_url(request, course_id, user=None, return_to=None, return_to_url=None):
```

Comment:
I don't necessarily think so - the current `return_to` expects a URL name and uses the course ID as an arg to reverse it; something like this might be an option:

```python
def get_enterprise_consent_url(request, course_id, user=None, return_to=None, course_specific_return=True):
    pass
```

But I don't think that gets us much as compared to this construction.